README.md

```diff
@@ -1,33 +1,82 @@
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be informed by data.
 
-We are respectful of different skill sets and levels of ability.
->We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+_Behaviours_: 
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
 
-We believe we are empowered to meet the needs of people we encounter across the Guardian.
+**Empowerment**
 
->This behaviour is consistently encouraged across all levels of engineering.
+We believe that all of us want to - and can - make a difference to the Guardian.
 
-We believe that embracing technical innovation and excellence can help us make a difference
+_Behaviours_: 
+  - Meeting the needs of people we encounter across the Guardian is consistently encouraged across all levels of engineering.
+  - We trust engineers to build the right things, collaborating closely and transparently with other teams.
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
 
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
```

Comment:
This is your history not your behaviour

Reply:
I have rewrite to made it clearer this history illustrate the behaviour